app.js

```diff
@@ -190,11 +192,11 @@ fs.readFile(path.resolve(__dirname, 'config/ipbans.txt'), (err, data) => {
 		if (!data[i]) continue;
 		if (data[i].includes('/')) {
 			rangebans.push(data[i]);
-		} else if (!Users.bannedIps[data[i]]) {
-			Users.bannedIps[data[i]] = '#ipban';
+		} else if (!Punishments.bannedIps[data[i]]) {
+			Punishments.bannedIps[data[i]] = '#ipban';
 		}
 	}
-	Users.checkRangeBanned = Cidr.checker(rangebans);
+	Punishments.checkRangeBanned = Cidr.checker(rangebans);
 });
```

Comment:
No I wanted to do it but Zarel merged it already, oh well. I did plan to clean everything up tomorrow, including this. And no, I do not expect Zarel's input for everything, but this PR was something he specifically asked me to do so I do want his input on what he wants from the things he asked me to do.